src/Donors/Migrations/addPhoneColumn.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Give\Donors\Migrations;
+
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ */
+class addPhoneColumn extends Migration
```

Comment:
```suggestion
class AddPhoneColumn extends Migration
```

Reply:
Resolved in 078a7d30bf7b851381f77f1b4785a2a5e0315de7